packages/picasso/src/RatingThumbs/story/index.jsx

```diff
@@ -0,0 +1,31 @@
+import PicassoBook from '~/.storybook/components/PicassoBook'
+import { RatingThumbs } from '../RatingThumbs'
+
+const chapter = PicassoBook.connectToPage(page =>
+  page
+    .createChapter('Thumbs Rating')
+    .addExample('RatingThumbs/story/Default.example.tsx', {
+      title: 'Default',
+      description: 'Default behavior'
+    })
```

Comment:
```suggestion
    }) // picasso-skip-visuals
```
Without this comment, every story is automatically becoming a visual test. We are trying to get rid of these old tests and for new and refactored components we are trying to write cypress tests with Happo. Checkout `/cypress/components`